### DISCUSSION TITLE
Java to C#

### DISCUSSION THREAD

**Discussion by sap3r85** [prowide/prowide-iso20022#47] [Q&A] [answered]
Hello,

what would be the best way to convert this code to C#?
have anyone done this on that repository?

**Comment by zubri** [comment, marked as answer]
The model uses JAXB annotations for marshaling/unmarshalling, so I don't think a direct C# migration would work.
Also notice this project depends on some API from the Prowide Core library, so you would need to port that one as well. And also the  third-party library dependencies.
